app/src/main/kotlin/com/simplemobiletools/calendar/pro/fragments/WeekFragment.kt

```diff
@@ -390,6 +390,7 @@ class WeekFragment : Fragment(), WeeklyCalendar {
                 x = todayColumn.x - extraWidth / 2
                 y = minutes * minuteHeight - markerHeight / 2
             }
+            currentTimeView!!.bringToFront()
         }
```

Comment:
I would never use !! what if currentView is null?  I would put this inside the currentView!!.apply{} and replace the !! with ?.